libsolidity/interface/Exceptions.cpp

```diff
@@ -33,22 +33,22 @@ Error::Error(Type _type, SourceLocation const& _location, string const& _descrip
 	switch(m_type)
 	{
 	case Type::DeclarationError:
-		m_typeName = "Declaration Error";
```

Comment:
It is used in the error messages, so it does change the interface if someone is parsing for these substrings.